src/server/utils/dates.py

```diff
@@ -86,16 +86,22 @@ def time_values_to_ranges(values: Optional[TimeValues]) -> Optional[TimeValues]:
     e.g. [20200101, 20200102, (20200101, 20200104), 20200106] -> [(20200101, 20200104), 20200106]
     (the first two values of the original list are merged into a single range)
     """
+    logger = get_structured_logger('server_utils')
     if not values or len(values) <= 1:
+        logger.info("List of dates looks like 0-1 elements, nothing to optimize", time_values=values)
```

Comment:
Do we still want to shift most of these to `debug`?

Reply:
The plan is to do that after some metrics are gathered from prod